src/package/polyfill/overrideCustomElementDefine.ts

```diff
@@ -21,6 +21,14 @@ export function overrideCustomElementDefine() {
         ) {
             const registeredCustomElement = customElements.get(elementName);
 
+            // save and clear attribute so we are in control
+            impl.__observedAttributes = impl.observedAttributes;
```

Comment:
Dangerous, to prevent collision we could use `Symbol('observedAttributes')` but we need to remember the Symbol in the hmrCache because in a HMR the Symbol would be redefined.

Reply:
Yes, good suggestion.

Reply:
fixed